src/cbmc/cbmc_parse_options.cpp

```diff
@@ -580,25 +590,32 @@ bool cbmc_parse_optionst::set_properties()
 }
 
 int cbmc_parse_optionst::get_goto_program(
-  const optionst &options)
+  goto_modelt &goto_model,
+  const optionst &options,
+  const cmdlinet &cmdline,
+  messaget &log,
+  ui_message_handlert &ui_message_handler,
+  message_handlert &mh)
```

Comment:
`mh` and `ui_message_handler` will actually be the same in the context of `cbmc_parse_optionst`, and both of which you'd  get by doing `log.get_message_handler()`. What you wouldn't be able to get, though, is `ui_message_handler.get_ui()` so maybe pass that in addition to just a `messaget`?